tests/cases/conformance/types/typeRelationships/typeInference/genericContextualTypes2.ts

```diff
@@ -0,0 +1,41 @@
+// @strict: true
+// @noEmit: true
+
+type LowInfer<T> = T & {};
```

Comment:
Since those tests are using `LowInfer` - cc @RyanCavanaugh who was mentioning this as a somewhat viable and stable pattern in some of the issues